dts/bindings/usb/nxp,kinetis-usbd.yaml

```diff
@@ -22,6 +22,3 @@ properties:
 
     interrupts:
       category: required
-
-    clocks:
```

Comment:
why not set to optional @pizi-nordic ?

Reply:
@galak already moved a bunch of common stuff into `base.yaml`, which is included indirectly. `base.yaml` has this:

```
clocks:
    type: array
    category: optional
    description: Clock gate information
```